### PR TITLE
Feature/sn warnings

### PR DESCRIPTION
**Pull request by hroest**
allow people to turn off the SNE warnings and provides an interface to read out inside the program to see what went wrong (to potentially adjust parameters)

### CODE REVIEW

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h [hunk 1]
```diff
@@ -153,26 +155,37 @@ namespace OpenMS
 
     //@}
 
-
     /// Destructor
     virtual ~SignalToNoiseEstimatorMedian()
     {}
 
+    /// Returns how many percent of the windows were sparse
+    double getSparseWindowPercent()
```
Comment (cbielow) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h]: const
(same below)

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h [hunk 2]
```diff
@@ -55,13 +55,13 @@ namespace OpenMS
     The whole computation is histogram based, so the user will need to supply a number of bins (param: <i>bin_count</i>), which determines
     the level of error and runtime. The maximal intensity for a datapoint to be included in the histogram can be either determined
     automatically (params: <i>AutoMaxIntensity</i>, <i>auto_mode</i>) by two different methods or can be set directly by the user (param: <i>max_intensity</i>).
-    If the (estimated) <i>max_intensity</i> value is too low and the median is found to be in the last (&highest) bin, a warning to std:err will be given. In this case you should increase
```
Comment (cbielow) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h]: why remove this?

File: src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakPickerMRM.h [hunk 3]
```diff
@@ -121,24 +121,40 @@ namespace OpenMS
     PeakPickerMRM& operator=(const PeakPickerMRM& rhs);
 
     // Members
+    /// Frame length for the SGolay smoothing
     UInt sgolay_frame_length_;
+    /// Polynomial order for the SGolay smoothing
     UInt sgolay_polynomial_order_;
+    /// Width of the gaussian smoothing
```
Comment (cbielow) [inline src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakPickerMRM.h]: Gaussian
(same below)

**Comment by cbielow** [comment]
only minor stuff,
+1

File: src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h (hunk 2 above)
Comment (hroest) [inline src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h]: I thought it somewhat redundant with the lines below but we can also keep it

**Comment by hroest** [comment]
ok, addressed all the comments

**Pull request merged**